src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -876,25 +878,15 @@ async def ModelInfo(
 
 
 async def start_grpc_server(
-    engine: AsyncLLMEngine, args: argparse.Namespace
+    engine_client: AsyncEngineClient,
+    args: argparse.Namespace,
 ) -> aio.Server:
-    # Log memory summary after model is loaded
-    from torch.cuda import memory_summary
-
-    assert isinstance(engine, AsyncLLMEngine)
-    assert isinstance(engine.engine, _AsyncLLMEngine)
-
-    if (device_type := engine.engine.device_config.device.type) == "cuda":
-        logger.info(memory_summary(engine.engine.device_config.device))
-    else:
-        logger.warning("Cannot print device usage for device type: %s", device_type)
```

Comment:
We cannot print the memory summary anymore, as this method is not available from the engine client.